frontend/pages/SoftwarePage/SoftwareTitles/SoftwareTable/SoftwareTable.tsx

```diff
@@ -284,28 +285,36 @@ const SoftwareTable = ({
             }
           />
         )}
+        <Slider
+          value={showVersions}
+          onChange={handleShowVersionsToggle}
+          inactiveText="Show versions"
```

Comment:
Just checked and every other place we're using the slider we we have different active vs. inactive text. Probably doesn't make sense here because this is a filter rather than a setting...which makes this feel like it should be a checkbox rather than a slider?

Out of scope for this revision but something to keep in the back of your mind (and [I added a Slack thread in #help-design](https://fleetdm.slack.com/archives/C02A8BRABB5/p1736368662760569) on this)